src/SprykerEco/Zed/Payone/Persistence/PayoneRepository.php

```diff
@@ -67,4 +187,60 @@ protected function setPayoneOrderItemFilters(
 
         return $paymentPayoneOrderItemQuery;
     }
+
+    /**
+     * @param int $idOrder
+     *
+     * @return \Orm\Zed\Payone\Persistence\SpyPaymentPayoneQuery
+     */
+    protected function createPaymentByOrderId(int $idOrder): SpyPaymentPayoneQuery
+    {
+        $query = $this->getFactory()->createPaymentPayoneQuery();
+        $query->findByFkSalesOrder($idOrder);
+
+        return $query;
+    }
+
+    /**
+     * @param \ArrayObject|\Generated\Shared\Transfer\OrderTransfer[] $orders
+     *
+     * @return \Orm\Zed\Payone\Persistence\SpyPaymentPayoneApiLogQuery
+     */
+    protected function createApiLogsByOrderIds($orders): SpyPaymentPayoneApiLogQuery
```

Comment:
* `createPaymentPayoneApiLogQueryByOrderIds()`.

Reply:
++

Reply:
Not done.

Reply:
++